src/server/api/routes/users.js

```diff
@@ -127,14 +131,17 @@ router.post('/users/login', async (req, res, next) => {
 
     const userDoc = await userCollection.doc(user).get();
     const isNew = !userDoc.exists;
-    await userCollection.doc(user).set(
-      {
-        user: userData,
-        accessToken,
-        refreshToken,
-      },
-      { merge: true }
-    );
+
+    const payload = {
+      user: userData,
+      accessToken,
+      refreshToken,
+    };
+    if (isNew) {
+      payload.followedUsers = FieldValue.arrayUnion(...DEFAULT_FOLLOW_IDS);
```

Comment:
`isNew` means `userDoc` does not exist, does array union work in this case, and is it needed?

Reply:
https://cloud.google.com/firestore/docs/manage-data/add-data#data_types

Seems plain array should be fine? 🤨

Reply:
It works for non exist, but yes actually not needed